tasks/configuration.yml

```diff
@@ -249,6 +249,24 @@
       inventory_hostname == icinga2_client_monitoring_parents[0]
       and not ('icinga2_client_director_managed' in hostvars[item] and hostvars[item].icinga2_client_director_managed)
 
+# @TODO: icinga2_master_client_parent_zone is not being created if no satelitte is being used THIS IS A WORKAROUND!
```

Comment:
I needed to use this workaround so my master zone gets created. Im not sure if i used the role wrong or if this is in fact necessary.